tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -48,6 +62,19 @@ const userSharesFileOrFolderWithUser = function (file, sharee, role) {
 const userSharesFileOrFolderWithGroup = function (file, sharee, role) {
   return userSharesFileOrFolderWithUserOrGroup(file, sharee, true, role)
 }
+
+Given('user {string} from remote server has shared {string} with user {string} from local server', function (sharer, file, receiver) {
+  receiver = util.format('%s@%s', receiver, backendHelper.getBackendUrl(BACKENDS.local))
+  return backendHelper.runOnRemoteBackend(
+    shareFileFolder, [file, sharer, receiver, SHARE_TYPES.federated_cloud_share]
+  )
+})
+
+Given('user {string} from local server has shared {string} with user {string} from remote server', function (sharer, shareServer, file, receiver, receiverServer) {
```

Comment:
this line is too long

Reply:
I don't think there is much we can do in these step definitions because the steps themselves can get too long.

Reply:
But, the function can be split into next line. :slightly_smiling_face: